components/Overlay/Overlay.tsx

```diff
@@ -0,0 +1,18 @@
+import { styled, css } from '../../stitches.config';
+
+export const overlayStyles = css({
```

Comment:
On my tests, I was able to select text behind the Overlay when it was open. I think we should prevent that in some way, maybe using pointer-events on the overlay CSS, or by closing the Overlay when it's clicked. WDYT? 

https://user-images.githubusercontent.com/38889364/144855412-f2d62cba-d3f0-4cdf-9afa-1d41d629fa03.mov

Reply:
@paulocfjunior the `Overlay` component is not here to reimplement the `Overlay` features coming from `Dialog`'s overlay.

It's just there to display the styles we apply and make sure those styles, that we reuse are working as expected. Therefore, the feature you request seems unrelated.

Maybe the current story is not clear enough. Please tell me if you have ideas or opinions on how to make clear this story is there simply to display `Overlay` basics without the radix primitive layer